### PR TITLE
reduce verbosity of license check message if >30 days to run

### PR DESCRIPTION
**Pull request by JerryShea**

### CODE REVIEW

File: src/main/java/net/openhft/chronicle/core/LicenceCheck.java [hunk 1]
```diff
@@ -34,32 +32,46 @@ public interface LicenceCheck {
...
+        licenceExpiry(product, caller, logLicenceExpiryDetails);
+    }
+
+    /**
+     * Provide licence expiry details
+     * @param product product
+     * @param caller caller
+     * @param licenceExpiryDetails callback to call with license days to run and license owner
+     */
+    static void licenceExpiry(String product, Class<?> caller, BiConsumer<Long, String> licenceExpiryDetails) {
```
Comment (JerryShea) [inline src/main/java/net/openhft/chronicle/core/LicenceCheck.java:64]: for https://github.com/ChronicleEnterprise/Chronicle-FIX/issues/1687

Comment (alamar) [reply src/main/java/net/openhft/chronicle/core/LicenceCheck.java:64]: Does it add property checking? I don't see that

**Review by alamar — APPROVED** [review]

File: src/main/java/net/openhft/chronicle/core/LicenceCheck.java (hunk 1 above)
Comment (JerryShea) [reply src/main/java/net/openhft/chronicle/core/LicenceCheck.java:64]: It doesn't. This change a) reduces verbosity and b) provides the licenseExpiry method so a consumer can find out how many days remaining

Comment (alamar) [inline src/main/java/net/openhft/chronicle/core/LicenceCheck.java:64]: Sonar says caller arg may be removed

Comment (JerryShea) [reply src/main/java/net/openhft/chronicle/core/LicenceCheck.java:64]: Yes but it is also an argument to the `check` method and so passed through. Assume we may use called in the future

**Pull request merged**